extensions/api/control/src/main/java/org/eclipse/dataspaceconnector/api/control/ControlApiServiceExtension.java

```diff
@@ -31,7 +31,8 @@
 
     @Override
     public Set<String> requires() {
-        return Set.of("edc:webservice", "dataspaceconnector:transfer-process-manager", "dataspaceconnector:dispatcher");
+        return Set.of("edc:webservice", "dataspaceconnector:transfer-process-manager",
+                "dataspaceconnector:dispatcher", "edc:core:contract");
```

Comment:
Can we replace this with a constant?

Reply:
Moved to a constant in ContractNegotiationManager. Should be adjusted in a another PR.